frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -727,6 +766,72 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
+		/// Submit a solution for the signed phase.
+		///
+		/// The dispatch origin fo this call must be __signed__.
+		///
+		/// The solution is potentially queued, based on the claimed score and processed at the end
+		/// of the signed phase.
+		///
+		/// A deposit is reserved and recorded for the solution. Based on the outcome, the solution
+		/// might be rewarded, slashed, or get all or a part of the deposit back.
+		///
+		/// # <weight>
+		/// Queue size must be provided as witness data.
+		/// # </weight>
+		#[pallet::weight(T::WeightInfo::submit(*witness_data))]
+		pub fn submit(
```

Comment:
This will needlessly bump the tx-version, could re-order to maintain the call backward compatibility.

Reply:
totally agree, this should be a the bottom

Reply:
OK: 725cadfe6